Data/ByteString.hs

```diff
@@ -40,8 +40,9 @@
 
 module Data.ByteString (
 
-        -- * The @ByteString@ type
+        -- * The strict @ByteString@ type and its associated type alias
```

Comment:
I'm afraid this is too long for a side bar table of contents in Haddock, it takes 5 lines. 
```suggestion
        -- * Strict @ByteString@
```